### PR TITLE
v2.0.0 release

### PR DESCRIPTION
**Pull request by fhunleth**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,85 +1,44 @@
 # Changelog
 
-## v2.0.0-pre.6 - 2024-1-9
+## v2.0.0 - 2024-01-10
```
Comment (jjcarstens) [inline CHANGELOG.md:3]:
```suggestion
## v2.0.0 - 2024-01-11
```
Just assuming this isn't merged/releasing until tomorrow

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,85 +1,44 @@
...
 
-This release only has naming changes and documentation updates.
+This is a major update to Circuits.GPIO that moderizes the API, restricts usage
```
Comment (jjcarstens) [inline CHANGELOG.md:5]:
```suggestion
This is a major update to Circuits.GPIO that modernizes the API, restricts usage
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,85 +1,44 @@
...
 
-## v2.0.0-pre.1 - 2023-12-29
+  * `Circuits.GPIO.open/3` is much more flexible in how GPIOs are identified.
+    Specifying GPIOs by number still works, but it's now possible to specify
+    GPIOs by string labels and by tuples that contain the GPIO controller name
+    and index. See `t:gpio_spec/0` and the `README.md` for details.
 
-This is a major update over pre.0 that has a significant upgrade to the backend
-for Nerves and Linux users.
+  * List out`available GPIOs with Circuits.GPIO.enumerate/0`. Other helper
```
Comment (jjcarstens) [inline CHANGELOG.md:21]:
```suggestion
  * List out available GPIOs with `Circuits.GPIO.enumerate/0`. Other helper
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,85 +1,44 @@
...
+    Nerves
 
-## v2.0.0-pre.0 - 2023-05-30
+  * Easily do one-off reads and writes with `Circuits.GPIO.read_one/2` and
+    `Circuits.GPIO.write_one/3`
 
-This is a major update to Circuits.GPIO that removes the requirement to use
-Nerves or Linux. The API is almost the same and the default is to compile and
-use the Linux backend, so changes may not be needed.
+  * Improved performance on Nerves and Linux and  kernel-applied timestamping of
```
Comment (jjcarstens) [inline CHANGELOG.md:30]:
```suggestion
  * Improved performance on Nerves, Linux, and kernel-applied timestamping of
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,85 +1,44 @@
...
+    GPIO input events
 
-This is a prerelease so APIs may still change before the v2.0.0 release.
+  * Add `Circuits.GPIO.Diagnostics` to automate runtime testing
 
 * Changes
-  * Support alternative backends for different operating systems or for
-    simulated hardware
-  * Defer loading the Linux NIF until `Circuits.GPIO.open/2` is called
+  * Workaround reordered GPIO controllers on Beaglebones with Linux 5.15 and
```
Comment (jjcarstens) [inline CHANGELOG.md:36]: This was in a previously release, correct? Does it need to be highlighted again

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,85 +1,44 @@
...
+    later. (Primarily for consistent pin numbers)
+  * More consistent error returns. Unexpected errors return `{:errno, value}`
+    tuples to help correlate errors to low level docs
+  * Deferred loading of the NIF so simplify debugging of GPIO backends.
```
Comment (jjcarstens) [inline CHANGELOG.md:40]:
```suggestion
  * Deferred loading of the NIF to simplify debugging of GPIO backends.
```

**Comment by fhunleth** [comment]
@jjcarstens Thanks for all of the fixes. I accepted them all.

**Review by jjcarstens — APPROVED** [review]

**Pull request merged**